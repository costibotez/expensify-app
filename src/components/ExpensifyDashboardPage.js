import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpensifyDashboardPage = () => (
    <div>
        <ExpenseListFilters/>
        <p>This is coming from my dashboard</p>
        <ExpenseList/>
    </div>
);

export default ExpensifyDashboardPage;