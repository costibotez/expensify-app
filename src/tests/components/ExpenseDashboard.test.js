import React from 'react';
import ExpensifyDashboardPage from '../../components/ExpensifyDashboardPage';
import { shallow } from 'enzyme';

test('should render ExpenseDashboard', () => {
    const wrapper = shallow(<ExpensifyDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
});