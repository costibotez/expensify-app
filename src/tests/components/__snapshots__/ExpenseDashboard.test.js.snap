// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboard 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <p>
    This is coming from my dashboard
  </p>
  <Connect(ExpenseList) />
</div>
`;
